contributing/documentation/overview.rst

```diff
@@ -107,6 +107,12 @@ your project:
     upstream  https://github.com/symfony/symfony-docs.git (fetch)
     upstream  https://github.com/symfony/symfony-docs.git (push)
 
+Fetch the upstream branch list.
```

Comment:
K. I'll reword and commit. Should I also squash the reword commit together
with the original?

On Monday, July 4, 2016, Christian Flothmann <notifications@github.com>
wrote:

> In contributing/documentation/overview.rst
> <https://github.com/symfony/symfony-docs/pull/6704#discussion_r69479011>:
>
> > @@ -107,6 +107,12 @@ your project:
> >      upstream  https://github.com/symfony/symfony-docs.git (fetch)
> >      upstream  https://github.com/symfony/symfony-docs.git (push)
> >
> > +Fetch the upstream branch list.
>
> And yeah, I agree with your rewording suggestion.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/symfony/symfony-docs/pull/6704/files/a8c589e3716b9a53afd065b1c88eeaab466fa781#r69479011>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVQZwdCKcN950V3I6Wj43bZt5kx4SJWks5qSTgtgaJpZM4JDk6s>
> .
>


-- 

-- Chance
email: chance@chancegarcia.com
portfolio:  http://chancegarcia.com/resume/portfolio